### DISCUSSION TITLE
json files not supported for flag -v

### DISCUSSION THREAD

**Discussion by ghost** [artilleryio/artillery#970] [Artillery]
Hello,

Are json files supported when using the -v cli flag?

currently im getting an error of "Variable definition is not valid JSON. Correct example: -v '{"var1": "value1", "var2": "value2"}'"
when i try and pass in a json file instead of manually typing in the long variable list.

Maybe im configuring this wrong?

 I have a json file with the following format
 ```
{
    "cCN":"Party", 
    "sECN":"Sin"
}
 ```
 
 also doing this through powershell if that makes a difference

**Comment by hassy** [comment]
The value of `-v` is expected to be a JSON string, rather than a file name. Not sure about Powershell, but in `bash`/`zsh` you could pass the contents of a file on the command line with shell substitution, e.g.:

```
artillery -v "$(cat somefile.json)"
```

but you'd have to make sure there are no line breaks.

I imagine Powershell has something similar.
